### DISCUSSION TITLE
2022 Eco Zurich Model with 100101, no function after power down

### DISCUSSION THREAD

**Discussion by AndyMu97** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#561] [Q&A]
Hi all,

after some experimenting I got my setup working. I’m an absolute rookie and learned a lot the last days 😁 thanks to all contributing on this project and make it happen! 
@visualapproach great job done!

All is doing more or less well, just one issue: if I power down the spa and start again, my display has no functionality, and the esp is not reconnecting to Wi-Fi.
I have to open the case, unplug everything and reconnect starting with the CIO connection. After this esp with wifi comes up and I can reconnect DISP. Display comes up and all is fine.
This is a workaround, but nothing I want to live with.
Any ideas?
I do have ESP Lolin nodeMCU V3 (by az-delivery in Germany) with the blue LLC, connectors on H1A and H2A.
HW setting is 6 wire, 2021 and PCB on V2B. Cross Cabling following page 9 in the instructions.

Any ideas how to solve?
Thanks a lot! Andy

**Comment by AndyMu97** [comment]
What if I change the power supply for the board to the onboard USB micro instead of getting the 5V via Vin by the pump? Would keep the board up and running permanently.

**Reply by visualapproach** [reply]
There is a risk with two power supplies. If they are connected somewhere and they are not exactly the same voltage you will have a potentially large current between them. I don't know how the usb is connected to Vin but that should be considered before doing it permanently.
So maybe it will work by having a permanent 5V supply and not using the 5V from the pump at all.

**Reply by AndyMu97** [reply]
Yes, I changed my first idea because this was not "professional".
The one or the other 5V support, not both. Will be done by only connecting the relevant Dx pins to the PBC (not the Vin/G) and use the USB micro onboard for power.

**Reply by visualapproach** [reply]
You must have a common GND. (Connect G to everything)

**Reply by AndyMu97** [reply]
Ok, but might this become a problem? As I have to use the G of the onboard USB, and it will be connected with the G coming via the pump cabling.

Well... I'd prefer to get the "not starting" issue solved instead of this workaround :-)

**Reply by visualapproach** [reply]
Well, I can't take responsibility for what will happen but if you don't connect the grounds the signals will likely not work. One GND might be 20V above or below the other GND. The "5V" is in relation to GND, and if you have different GND anything can happen.

**Comment by AndyMu97** [comment]
Well... switching over to red LLC which arrived today by parcel service... everything is now working!!!
Thanks @visualapproach for your support, I will donate a coffee :-)

**Reply by visualapproach** [reply]
Yay that's good news. Wouldn't guessed it was the LLC from the problem description but glad it's sorted! Thank you for supporting the project, I appreciate it!
